pkgs/by-name/sw/switcheroo/package.nix

```diff
@@ -48,11 +49,18 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     glib
     gtk4
+    imagemagick
```

Comment:
does this program also link against imagemagick libraries?

Reply:
No, thanks, I have removed it